lms/templates/footer.html

```diff
@@ -77,3 +77,7 @@
     <link rel="stylesheet" type="text/css" href="${url}"></link>
   % endfor
 % endif
+
+<!-- Start of philanthropyu Zendesk Widget script -->
+<script id="ze-snippet" src="https://static.zdassets.com/ekr/snippet.js?key=19c14808-6ad4-41f1-b5af-fcf69742adf8"></script>
```

Comment:
Can we add key value from deployment or site config?

Reply:
done